frontend/src/components/maps/leaflet/Map.test.tsx

```diff
@@ -28,6 +28,8 @@ const mockParcels = [
 jest.mock('hooks/useApi');
 jest.mock('hooks/pims-api');
 
+const onPropertyMarkerClick = jest.fn();
```

Comment:
are the tests not done for this? it seems odd to add a mocked function but never verify it, reset it, etc.

Reply:
not at the moment. Added it to fulfill the props for the component